.github/workflows/reusable-unit-split.yml

```diff
@@ -119,11 +119,10 @@ jobs:
           umask 0022
 
           go run gotest.tools/gotestsum@v${{env.GOTESTSUM_VERSION}} \
-          --format=short-verbose \
+          --format=github-actions \
+          --format-hide-empty-pkg \
           --jsonfile /tmp/jsonfile/go-test.log \
-          --debug \
-          --rerun-fails=3 \
-          --rerun-fails-max-failures=40 \
```

Comment:
The default of 2 reruns should be good enough. Flaky tests usually run their own retries as well and I'd rather deal with tests that can't pass in 2 tries.